README.md

```diff
@@ -22,6 +22,18 @@ mkcert -cert-file tls-cert.pem -key-file tls-key.pem localhost 127.0.0.1 ::1
 
 One caveat is HTTP redirect to HTTPS is not supported, you have to type in https in the browser address bar manually.
 
+## Prerequisite
+
+Note that there is a local .tool-version in project root. For the following setup to work, we need to
+
+1. Install asdf
+
+2. Run (please refer to the version in .tool-version)
+   ```sh
+   asdf install nodejs 12.18.3
+   asdf install golang 1.15
```

Comment:
Just run `asdf install` without any argument. It will pick up the content of `.tool-versions` in the current directory.